src/bundle/Resources/config/dashboard.yml

```diff
@@ -0,0 +1,87 @@
+services:
+    _defaults:
+        autowire: true
+        autoconfigure: true
+        public: false
+
+    EzSystems\HybridPlatformUi\Dashboard\:
+        resource: '../../../lib/Dashboard/*'
+
+    EzSystems\HybridPlatformUi\Dashboard\Dashboard:
+
+    EzSystems\HybridPlatformUi\Dashboard\Section\MyContent:
+        calls:
+            - [setTemplate, ['EzSystemsHybridPlatformUiBundle:dashboard:my_content_section.html.twig']]
+            - [setParameter, ['identifier', 'my_content']]
+        tags:
+            - name: ezplatform.ui.dashboard.section
+              identifier: my_content
+              position: 1
```

Comment:
if #92 ends up with `priority`, maybe this should as well so it's clear what it does.